presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -355,6 +361,12 @@ Property Name                             Description
 ``delete_mode``                            Optionally specifies the write delete mode of the Iceberg        ``merge-on-read``
                                            specification to use for new tables, either ``copy-on-write``
                                            or ``merge-on-read``.
+
+``metadata_previous_versions_max``         Optionally specifies the max number of previous metadata files   ``100``
+                                           recorded in current metadata.
+
+``metadata_delete_after_commit``           Optionally specifies whether enable to delete the oldest         ``false``
```

Comment:
```suggestion
``metadata_delete_after_commit``           Set to ``true`` to delete the oldest         ``false``
```
Similar to previous comments. Like the precious comments about ``iceberg.metadata-delete-after-commit``, please adjust my suggested text change to fit the formatting needed for the columns in the table.

Reply:
OK, done.